vulkan.py

```diff
@@ -159,11 +159,12 @@ def __repr__(self):
                 (self.ret, self.name, param_str)
 
 class Extension(object):
-    def __init__(self, name, headers, objects, protos):
+    def __init__(self, name, headers, objects, protos, ifdef = None):
```

Comment:
This file "vulkan.py" belongs to LVL (https://github.com/KhronosGroup/Vulkan-LoaderAndValidationLayers)
This patch needs to split and commit this file to LVL and vk-vtgenerate.py changes to VulkanTools